packages/cli/commands/worker.ts

```diff
@@ -259,6 +265,10 @@ export class Worker extends Command {
 			try {
 				const { flags } = this.parse(Worker);
 
+				if (flags.stopTime) {
```

Comment:
since a `default` is defined, we don't need to check this. We can remove `static stopTime = 30`, and just use `Worker.stopTime = flags.stopTime;`